src/Operators/remapping.jl

```diff
@@ -268,10 +268,29 @@ end
 
 nxelems(topology::Topologies.IntervalTopology) =
     Topologies.nlocalelems(topology)
-nxelems(topology::Topologies.Topology2D{<:ClimaComms.SingletonCommsContext}) =
+function nxelems(
+    topology::T,
+) where {
+    T <: Union{
+        Topologies.Topology2D{<:ClimaComms.SingletonCommsContext},
+        Topologies.Topology2D{<:ClimaComms.MPICommsContext},
+    },
+}
     size(Meshes.elements(topology.mesh), 1)
-nyelems(topology::Topologies.Topology2D{<:ClimaComms.SingletonCommsContext}) =
+end
+function nyelems(
+    topology::T,
+) where {
+    T <: Union{
+        Topologies.Topology2D{<:ClimaComms.SingletonCommsContext},
+        Topologies.Topology2D{<:ClimaComms.MPICommsContext},
+    },
```

Comment:
```suggestion
    T <: Topologies.Topology2D{<:ClimaComms.AbstractCommsContext}
```
Maybe just use the abstract comms context here if it applies?